e2e-tests/recover/seedPhrase.spec.js

```diff
@@ -0,0 +1,44 @@
+const { createTestSubaccount } = require("../utils/account");
+
+describe("Account Recovery Using Seed Phrase", () => {
+    let testAccount;
+    beforeAll(async () => {
+        testAccount = await createTestSubaccount();
+    });
+
+    afterAll(async () => {
+        await browser.close();
+        await testAccount.delete();
+    });
+
+    it("navigates to seed phrase page successfully", async () => {
+        const user = await page.goto("/").then(() => page.getDocument());
+        await user
+            .findByRole("button", {
+                name: "Import Existing Account",
+            })
+            .then((importAccountBtn) => importAccountBtn.click())
+            .then(() => user.findByText("Recover Account"))
+            .then((recoverAccountBtn) => recoverAccountBtn.click());
```

Comment:
Rather than use specific name + roles, I want use to put _explicit_ ids on the elements we need to find with these tests; this is brittle as it currently stands for a few reasons:
1. If we rename the user-facing text, our tests will fail
2. If we run tests in an alternative language, the text string will differ depending on which language we are running the tests in)
3. If we change the _type_ of an element (e.g. refactor a div to a button) we don't want our tests to fail

https://playwright.dev/docs/selectors#id-data-testid-data-test-id-data-test-selectors

This has the added benefit of making our searches _consistent_ -- we can always use a simple selector to find the element by `data-test-id` and dispatch the `click()` directly to it without any temporary local variables -- so this would become:

```javascript
await page.click('data-test-id=importExistingAccountButton');
await page.click('data-test-id=recoverAccountButton');
expect(page).toMatchURL(/\/recover-seed-phrase$/);
```

Reply:
I think this is a (semi case by case) design decision for when we want our tests to fail but my understanding is that we want to simulate user behaviour as closely as possible when writing the tests (not just e2e). We should still use test ids for elements that we reference often or that determine critical state (e.g. currently logged in user). 

> If we rename the user-facing text, our tests will fail

We might want this to be the behaviour of our tests or to make assertions on this since the user facing result has changed.

> If we run tests in an alternative language, the text string will differ depending on which language we are running the tests in)

We might have to import localized strings if we want to do e2e tests in multiple languages. 

> If we change the type of an element (e.g. refactor a div to a button) we don't want our tests to fail

We could make it a practice to match by text in general rather than role.

I also think too many test ids will cause significant maintenance overhead. 

This is an important point for how we write all our tests so we should decide on guidelines on this.